runtime/onert/backend/cpu/ops/FullyConnectedLayer.cc

```diff
@@ -164,13 +164,17 @@ void FullyConnectedLayer::run()
 void FullyConnectedLayer::prepare()
 {
 #ifdef USE_RUY_GEMV
+  // TODO This is workaround
   // The only fc hybrid will use ruy kernel
   if (_input->data_type() != OperandType::FLOAT32 ||
       _weights->data_type() != OperandType::QUANT_INT8_SYMM)
   {
     return;
   }
 
+  // For now, dynamic input cannot support
+  assert(_input->is_dynamic() == false);
```

Comment:
Why did you add this as an assertion?
If `_input` is dynamic, this method is not called?

Reply:
Is this your comment's intention? I understood that this fc doesn't support dynamic tensor as input. Exactly what does it 
mean?

> It should also check if the input is dynamic.

Reply:
FYI I'm not good at dynamic tensor code.

Reply:
If so, could you add a TODO comment instead of the assertion.